ros2topic/ros2topic/verb/list.py

```diff
@@ -33,19 +45,31 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--include-hidden-topics', action='store_true',
             help='Consider hidden topics as well')
+        parser.add_argument(
+            '-v', '--verbose', action='store_true',
+            help='list full details about each topic')
 
     def main(self, *, args):
-        with NodeStrategy(args) as node:
+        topic_info = []
+        with DirectNode(args) as node:
```

Comment:
Using a`DirectNode` here instead of a `NodeStrategy` will not use the darlin and therefore result in either a longer time to wait for discovery or incomplete information. So please using the `NodeStrategy` here.

Reply:
I use `NodeStrategy` instead now. However, I need to register `count_publishers` and `count_subscribers` to use these functions, so I also modified the daemon.